runtime/doc/builtin.txt

```diff
@@ -1857,10 +1857,15 @@ complete_info([{what}])				*complete_info()*
 				See |complete_info_mode| for the values.
 		   pum_visible	|TRUE| if popup menu is visible.
 				See |pumvisible()|.
-		   items	List of completion matches.  Each item is a
-				dictionary containing the entries "word",
+		   items	List of all completion candidates. Each item
+				is a dictionary containing the entries "word",
 				"abbr", "menu", "kind", "info" and "user_data".
 				See |complete-items|.
+		   matches	Same as "items", but only returns items that
+				are shown in the |popup-menu|. If both "matches"
```

Comment:
by "shown in the popup menu" do you mean "matching the current query" ? because those 2 are subtly different.

The menu scrolls, so you could view 'shown' as just the ones that are on-screen, whereas I think you mean all that match the query.